dev/CommonStyles/AppBarButton_themeresources.xaml

```diff
@@ -413,12 +418,13 @@
                                 AutomationProperties.AccessibilityView="Raw" />
                             <FontIcon x:Name="SubItemChevron"
                                 Grid.Column="2"
-                                Glyph="&#xE0E3;"
+                                Glyph="{ThemeResource AppBarButtonFlyoutGlyph}"
```

Comment:
You could change to StaticResource

Reply:
> You could change to StaticResource

Yea, the glyphs typically wont change between Light, Dark, High Contrast modes, only colours

Reply:
This is intentional. ThemeResource is like saying "no matter where you override this, it will definitely work" and StaticResource is a lot less certain.

Reply:
The {StaticResource} markup extension does not restart its lookup from the originating location, it instead resumes the lookup from the current walk.  Since this {StaticResource} is in a control template that will be included in App.xaml it means that using static resource will cause this template to check visual tree element's resources for overrides.